File: app/src/main/java/com/levibostian/androidblanky/activity/MainActivity.java
package com.levibostian.androidblanky.activity;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import com.levibostian.androidblanky.R;
import com.levibostian.androidblanky.fragment.MainFragment;

public class MainActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        startupFragment();
    }

    private void startupFragment() {
        getFragmentManager().beginTransaction()
                .add(R.id.fragment_container, MainFragment.newInstance())
                .commit();
    }

}

File: app/src/main/java/com/levibostian/view/ui/activity/MainActivity.kt
package com.levibostian.view.ui.activity

import android.content.Context
import android.content.Intent
import android.os.Bundle
import androidx.appcompat.widget.Toolbar
import androidx.navigation.NavController
import androidx.navigation.findNavController
import androidx.navigation.ui.AppBarConfiguration
import androidx.navigation.ui.setupWithNavController
import com.levibostian.R
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity: BaseActivity() {

    private val navController: NavController
        get() = findNavController(R.id.nav_host_fragment)

    companion object {
        fun getIntent(context: Context): Intent = Intent(context, MainActivity::class.java)
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)

        setSupportActionBar(toolbar as Toolbar)
        (toolbar as Toolbar).setupWithNavController(navController, AppBarConfiguration(navController.graph))
    }

}